neurogym/envs/contextdecisionmaking.py

```diff
@@ -81,51 +82,29 @@ def __init__(
         if timing:
             self.timing.update(timing)
 
-        self.abort = False
+        self.abort = abort
```

Comment:
I see that throughout the package, `self.abort` is always False with no way to change it as a user. Should we change it throughout?
Also, now that it's an outward facing class attribute, maybe it would be better to give it a clearer name. Now, as a user I might get confused and think it immediately aborts the trial or upon any mistake or sth. Maybe something like `strict_fixation`?